src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,32 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE_FORMATTER = "dd.MM.yyyy";
+    private static final String DATA_LINE_SEPARATOR = " ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        System.out.println();
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(DATE_FORMATTER);
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, dateTimeFormatter);
+        LocalDate localDateTo = LocalDate.parse(dateTo, dateTimeFormatter);
+        String[] returnStrings = new String[names.length];
+        StringBuilder returnString = new StringBuilder("Report for period "
```

Comment:
if you are using StringBuilder you don't need to use concatenation. 

bad naming - result will be better